sdk/communication/Azure.Communication.CallAutomation/tests/CallAutomationClient/CallAutomationClientLiveTests.cs

```diff
@@ -122,8 +122,8 @@ public async Task CreateCallToPSTNGetCallAndHangUpCallTest()
         [Test]
         public async Task CreateCallWithMediaStreaming()
         {
-            //if (SkipCallingServerInteractionLiveTests)
-            //    Assert.Ignore("Skip callingserver interaction live tests flag is on.");
+            if (SkipCallAutomationInteractionLiveTests)
+                Assert.Ignore("Skip callingserver interaction live tests flag is on.");
```

Comment:
nit: just a small thing, maybe change the description from callingserver to CallAutomation like the others